co2calculator/calculate.py

```diff
@@ -358,32 +361,35 @@ def calc_co2_heating(
     """
     # Set defaults
     if unit is None:
-        unit = "kWh"
+        unit = Unit.KWH
         warnings.warn(f"Unit was not provided. Assuming default value: '{unit}'")
     if area_share > 1:
         warnings.warn(
             f"Share of building area must be a float in the interval (0,1], but was set to '{area_share}'\n."
             f"The parameter will be set to '1.0' instead"
         )
+    if fuel_type is None:
+        fuel_type = HeatingFuel.GAS
+        warnings.warn(
+            f"No fuel type specified. Using default value: '{fuel_type}'"
+        )
     valid_unit_choices = ["kWh", "l", "kg", "m^3"]
     assert (
-        unit in valid_unit_choices
+            unit in valid_unit_choices
     ), f"unit={unit} is invalid. Valid choices are {', '.join(valid_unit_choices)}"
     if unit != "kWh":
```

Comment:
Does this work or does it have to be `unit != Unit.KWH` since the `Unit` enum has been added?

Reply:
thanks, I tried to fix it :) Works both ways but cleaner with unit != Unit.KWH
I also changed the `valid_unit_choices` so that these are retrieved from the Unit enum :)

Reply:
Got mail notifications for this one and was curious :) 

Two cents from my side:
Python enums work (and should be used) with 'is' and 'is not'.